detail/provisioning.cattle.io.cluster.vue

```diff
@@ -344,10 +393,85 @@ export default {
         elem:      event.target
       });
     },
+
     showPoolActionButton(pool) {
       return !!pool.availableActions?.length;
     },
 
+    async connectLog() {
+      if ( this.logSocket ) {
+        await this.logSocket.disconnect();
+        this.logSocket = null;
+      }
+
+      const params = {
+        follow:     true,
+        timestamps: true,
+        pretty:     true,
+      };
+
+      let url = this.value.mgmt?.linkFor('log');
+
+      url = addParams(url.replace(/^http/, 'ws'), params);
+
+      this.socket = new Socket(url, true, 0);
```

Comment:
These should all be `socket` --> `logSocket` to make use of the disconnect handling

Reply:
Oops.. thx